scripts/release/change_management

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+const REQUEST_PROMISE = require('promise-request-retry')
+
+// Set these in CircleCI in the heroku/cli project
+//
+const RELEASE_ACTOR_EMAIL = process.env.TPS_API_RELEASE_ACTOR_EMAIL
+const TPS_API_APP_ID = process.env.TPS_API_APP_ID
+const TPS_API_STAGE = process.env.TPS_API_STAGE || 'production'
+const TPS_API_TOKEN_PARAM = process.env.TPS_API_TOKEN_PARAM
+const TPS_API_URL_PARAM = process.env.TPS_API_URL_PARAM
+
+// This is set by CircleCI automagically
+//
+const RELEASE_COMMIT_SHA = process.env.CIRCLE_SHA1
+
+async function sendDeployNotification () {
+  const uri = TPS_API_URL_PARAM
+  const token = TPS_API_TOKEN_PARAM
+  const stage = TPS_API_STAGE
+  const sha = RELEASE_COMMIT_SHA
+  const appId = TPS_API_APP_ID
+  const email = RELEASE_ACTOR_EMAIL
+
+  const body = {
+    release: {
+      actor_email: email,
+      app_id: appId,
+      sha,
+      stage,
+      description: `Deploy ${sha} of heroku/cli in ${stage}`
+    }
+  }
+
+  try {
+    // Retry up to 6 (~2m) doubling the delay each time
+    //
+    const options = {
+      method: 'POST',
+      headers: {
+        Authorization: `Token ${token}`,
+        ACCEPT: 'application/json',
+        'Content-Type': 'application/json'
+      },
+      uri,
+      body,
+      accepted: [401, 403, 404],
+      retry: 6,
+      delay: 1000,
+      factor: 2,
+      json: true
+    }
+    const resp = await REQUEST_PROMISE(options)
+    console.log(resp)
+  } catch (error) {
+    console.log(error)
+    console.log(body)
+    process.exit(1)
+  }
+}
+
+sendDeployNotification()
```

Comment:
I'm not sure how it works with scripts, but do we need to `await` here? Maybe node knows not to exit until the promise has completed?

Reply:
```
function longPromise() {
	new Promise(resolve => {
		setTimeout(resolve, 4000);
	})
	.then(() => console.log('all done'));
}

longPromise();
```

Ran this as a script and node waited, worked on Node 10 and 12 👍

Reply:
`await` is only callable inside functions. Node won't exit until the event loop is done.